jspoon/src/main/java/pl/droidsonroids/jspoon/HtmlAdapter.java

```diff
@@ -95,6 +112,25 @@ public T fromInputStream(InputStream inputStream, URL baseUrl) throws IOExceptio
         return fromInputStream(inputStream, null, baseUrl);
     }
 
+    /**
+     * Populates {@code T} instance from the provided {@code inputStream}.
+     * <p>
+     * Does not close the {@code InputStream}.
```

Comment:
This one is copied from/based on your other method. Assuming lowercased inputStream referes to the argument name rather than java object.  Also usually objects in javadocs are mentioned with` {@link InputStream}` rather than `{@code inputStream}.`

Reply:
OK, we'll check and unify that.